src/Form/Extension/AddressTypeExtension.php

```diff
@@ -64,7 +64,12 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
     {
         $builder
             ->addEventListener(FormEvents::PRE_SET_DATA, function (FormEvent $event): void {
-                $billingAddressForm = $event->getForm()->getParent()->get('billingAddress');
+                $parentForm = $event->getForm()->getParent()
+                if (null === $parentForm) {
+                  return;
```

Comment:
Could you also use 4 spaces (tabulation) instead of two? Perhaps your IDE setting has 2 spaces for tab instead of 4. If you're using PHPStorm, read this: https://intellij-support.jetbrains.com/hc/en-us/community/posts/206332339-Tabs-and-Indents-are-2-spaces-despite-setting-at-4-spaces 🙂

Reply:
sorry, PR updated